Dockerfile.crust-builder

```diff
@@ -1,6 +1,6 @@
 FROM golang:1.12-alpine
 
-RUN	apk add --no-cache git make bash && \
+RUN	apk add --no-cache git make bash php7 php7-json php7-tokenizer && \
```

Comment:
fyi https://github.com/crusttech/crust/issues/111

Reply:
While that's a nice wish, rewriting codegens to go isn't a priority ATM. I'll leave the issue open, but it's going on low prio (we shouldn't be working on this right now)